compiler/x/codegen/OMRAheadOfTimeCompile.cpp

```diff
@@ -0,0 +1,131 @@
+/*******************************************************************************
```

Comment:
Given the work that @dsouzai has done in the meantime in OpenJ9, we should probably use the common version of this code rather than the x86-specific version. But I'll allow that as a cleanup activity rather than requiring it in this PR.

Reply:
Thank you for the comment and for allowing it for now. Hope it is not too late